### DISCUSSION TITLE
Can't rename node into cluster

### DISCUSSION THREAD

**Discussion by nuget1991** [rabbitmq/rabbitmq-server#3196] [Other]
Hi, I've updated cluster to latest 3.8.19 version and updated erlang of course to erlang-23.3.4.4-1.el7.x86_64. Before it I had 3.8.9 version and had known bug with renaming nodes(command didn't work properly). After upgrading rabbit on cluster with 3 nodes renaming command started working. But I still can't rename nodes. I have followed according to manual of official documentation https://www.rabbitmq.com/rabbitmqctl.8.html#rename_cluster_node
  So when I started rabbit node after renaming it still have the old name. What is correct way of renaming cluster node in latest version? I've already tried different ways: 
  stop node reset it, start
  remove all data from /var/lib/rabbitmq and start it again
  remove node from cluster and tried rename it etc
    Please help if you know the sollution. Thanks in advance!

**Comment by nuget1991** [comment]
any updates?

**Reply by HoloRin** [reply]
Hi @nuget1991 , can you provide logs and more detailed instructions for reproducing this issue? Node renames are part of the test suites run against every commit to this repository, so at least for those tested scenarios, we are confident things have been working correctly.
